### ISSUE TITLE
Black out tool also for generated PDF's?

### ISSUE THREAD

**Issue by postdigital** [DataRights/MAPPED#56]
I was thinking... Don't we need a black out tool **also** for generated PDF's (e.g. access requests and reminders)? What if someone wants to share on the forum his/her own access request letter containing some private data? 
So far we are only concerned about blacking out the received letters, not the ones we send.

**Comment by Rene-Mahieu** [comment]
I agree that this is needed.

**Comment by postdigital** [comment]
@mani47 + @shahriarb , is that doable?

**Comment by hadiasghari** [comment]
I agree we want the blackout tool also for the forum (this is also part of our data-sharing protocol in the ethics documents that users have control over what they share in forum).  Also we should provide a wikipage to guide the user in what kind of things to blackout.
